File: src/mg_webpage/models/project.clj
(ns mg-webpage.models.project
  (:use hiccup.core)
  (:require [mg-webpage.models.common :as common]))

(def projects-page
  (html [:html
         ;common/head
         [:head
          [:link {:href "/css/screen.css", :rel "stylesheet", :type "text/css"}]]
         [:meta {:name "viewport" :content "width=device-width, initial-scale=1.0"}]
         [:body
          common/navbar
          [:ul.projects
           [:li.project
            [:a.link {:href "http://alarmtube.marissagrove.com"} "Alarmtube-JS"]
            [:p#project "An alarm that plays YouTube videos when the alarm is triggered. Written in JavaScript."]]
           [:br]
           [:li.project
            [:a.link {:href "http://alarmtube.marissagrove.com"} "Alarmtube-CLJS"]
            [:p#project "An alarm that plays YouTube videos when the alarm is triggered. Written in ClojureScript."]]
           [:br]
           [:li.project
            [:a.link
             {:href "http://findaplace.marissagrove.com"}
             "Find a Place"]
            [:p#project "Platform for rating and reviewing businesses. Written in Python."]]]]]))

File: resources/public/js/components/Project.js
import React from 'react';

var Projects = React.createClass({
    render: function() {
        return (
	        <div>
	        <ul className="projects">
		<li className="project">
	        <a className="link" href="https://www.npmjs.com/package/react-embedded-twitter-timeline"> npm package react component for an embedded Twitter timeline</a>
	        <p className="project"> Install this npm package to embed your Twitter timeline in a react project, deals with operation order issues for you.</p>
	        </li>
	        <li className="project">
	        <a className="link" href="http://alarmtube.marissagrove.com"> Alarmtube-JS </a>
	        <p className="project"> An alarm that plays YouTube videos when the alarm is triggered. JavaScript</p>
	        </li>
	        <li className="project">
	        <a className="link" href="http://findaplace.marissagrove.com"> Find a Place </a>
	        <p className="project"> Platform for rating and reviewing businesses. Python</p>
	        </li>
	        </ul>
	        </div>
        );
    }
});

var exports = {
  handler: Projects
}

export default exports;	    
